configuration/concepts/program_attribute_types.csv

```diff
@@ -0,0 +1,35 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Description:es,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:700
+c969a930-b6f3-4b73-b93e-52defe1c9217,,,1,1,,,,,Misc,N/A,,,,,,,,,,,
+e0c53584-3e26-4edd-94d8-0f46b3333c49,,,2,2,,,,,Misc,N/A,,,,,,,,,,,
+01d56851-89c5-487b-836a-2783f826b1e7,,,3,3,,,,,Misc,N/A,,,,,,,,,,,
+347c7d54-0a11-4120-8016-b2e596ec75b8,,,4,4,,,,,Misc,N/A,,,,,,,,,,,
+d5a4c1c5-6356-445a-a42d-4c2cf69b0745,,,Upper extremity,Upper extremity,,,,,Misc,N/A,,,,,,,,,,,
+67d022ec-b598-4994-bcc3-1bc37eb5c2db,,,Lower extremity,Lower extremity,,,,,Misc,N/A,,,,,,,,,,,
+c8de1341-c048-4e92-8f3c-11469288b7ac,,,Orthopedic/Peds,Orthopedic/Peds,,,,,Misc,N/A,,,,,,,,,,,
+9296394c-dd38-4ea5-b7ac-d2d23ca9fae8,,,Sports Orthopedics,Sports Orthopedics,,,,,Misc,N/A,,,,,,,,,,,
+d6816557-c759-4280-bfea-f1d0c8a7aa86,,,Foot/ankle,Foot/ankle,,,,,Misc,N/A,,,,,,,,,,,
+01cb0485-6006-4f84-81f5-b1be4a91ef14,,,Podiatry,Podiatry,,,,,Misc,N/A,,,,,,,,,,,
+9661ccf0-88da-4e30-9e47-5e6ffdf044ef,,,Total Hip,Total Hip,,,,,Misc,N/A,,,,,,,,,,,
+0a0e05ad-5057-4929-b41d-e8a43341ec57,,,Total Knee,Total Knee,,,,,Misc,N/A,,,,,,,,,,,
+4416bc76-5969-4d6d-ac72-44d95feb20cd,,,Shoulder,Shoulder,,,,,Misc,N/A,,,,,,,,,,,
+9d11e6a5-56d5-4d6e-899e-4579af99cbfc,,,Spine,Spine,,,,,Misc,N/A,,,,,,,,,,,
+ac2184d7-98c5-4548-8f5c-ae8dc25bef39,,,Surgical Oncology,Surgical Oncology,,,,,Misc,N/A,,,,,,,,,,,
+ce6e838b-6a4c-474d-8278-c4314f7b6e25,,,General Surgery,General Surgery,,,,,Misc,N/A,,,,,,,,,,,
+da9cc967-ae63-4ac0-bf5e-b52eac04a820,,,ENT,ENT,,,,,Misc,N/A,,,,,,,,,,,
+8ef9c049-e03f-43ad-9225-81577bb13028,,,Urology,Urology,,,,,Misc,N/A,,,,,,,,,,,
+5dafe35b-94fb-4978-8f48-e92950f52c36,,,Gynecology,Gynecology,,,,,Misc,N/A,,,,,,,,,,,
+75bc6f17-d087-4c28-9f5a-da98ed3812d4,,,Ophthalmology,Ophthalmology,,,,,Misc,N/A,,,,,,,,,,,
+ee4ac5ca-7be1-44cb-a750-dba64720453a,,,Dental,Dental,,,,,Misc,N/A,,,,,,,,,,,
+1c64a247-fc10-4e12-982c-c73859ed3782,,,Family Medicine,Family Medicine,,,,,Misc,N/A,,,,,,,,,,,
+b2017b29-ee16-4c62-8d3e-5ff2c2d3ca07,,,Internal Medicine,Internal Medicine,,,,,Misc,N/A,,,,,,,,,,,
+bef52504-d77b-485c-ab81-9ccb6eb8f239,,,Pediatrics,Pediatrics,,,,,Misc,N/A,,,,,,,,,,,
+2484cdc7-a77a-4305-8bcc-b9c90f384bf6,,,Cardiology,Cardiology,,,,,Misc,N/A,,,,,,,,,,,
+dc04cfe1-29d1-46a7-a0cf-b8baacfdbb31,,,Dermatology,Dermatology,,,,,Misc,N/A,,,,,,,,,,,
+13aa8824-8a29-46c9-b80e-9201f03e82e5,,,Wound,Wound,,,,,Misc,N/A,,,,,,,,,,,
+eeed1828-3038-4997-ada8-55f75c0d33db,,,Other,Other,,,,,Misc,N/A,,,,,,,,,,,
```

Comment:
This concept already exists in the 'others.csv'. Let us not duplicate them

Reply:
Done

Reply:
@nuhuhmutebi  Change the file order such that 'Other.csv' is loaded first before this file

Reply:
Done